flow-decomposition/src/test/java/com/powsybl/flow_decomposition/FlowDecompositionResultsTests.java

```diff
@@ -147,7 +153,7 @@ void testBuilderN2State() {
         assertEquals(branchId, decomposedFlow.getBranchId());
         assertEquals(contingencyId3, decomposedFlow.getContingencyId());
         assertEquals(xnecId, decomposedFlow.getId());
-        assertEquals(10.0, decomposedFlow.getAcReferenceFlow());
+        assertEquals(10.0, decomposedFlow.getAcTerminal1ReferenceFlow());
```

Comment:
Can you add a test with terminal 2 as well please ?
`assertEquals(10.0, decomposedFlow.getAcTerminal2ReferenceFlow());`